tests/AppFixture.js

```diff
@@ -1,2 +1,3 @@
-baseline = /\d+ variables et \d+ paramètres référencés/
+baseline = /OpenFisca makes law computable. It is a free and open-source computation engine that allows modelling tax and benefit systems in a collaborative and transparent way. This software is used by researchers and public agencies./
+stats = /\d+,\d+ variables and \d+,\d+ parameters referenced/
```

Comment:
This assumes that there will always be over 1,000 variables and parameters. Rather use `[0-9,]+`.